src/widget/weffectparameternamebase.cpp

```diff
@@ -66,6 +78,11 @@ void WEffectParameterNameBase::parameterUpdated() {
         m_text = kNoEffectString;
         setBaseTooltip(tr("No effect loaded."));
     }
+    optimumWidth = math_max(
+            optimumWidth,
+            metrics.size(0, m_text).width());
+    setMinimumWidth(optimumWidth + 6);
```

Comment:
Why +6 ?
I tested with Tango & LateNight and it looks good without it, even with crammed effect slots, +6 only shifts the 'elide' threshold a bit. Is that it?